packages/dataviews/src/style.scss

```diff
@@ -387,16 +387,14 @@
 				border-radius: $grid-unit-05;
 			}
 		}
-		h3 {
-			overflow: hidden;
-			text-overflow: ellipsis;
-			white-space: nowrap;
```

Comment:
No longer needed as truncation is applied to `.dataviews-view-list__primary-field`.